src/Storefront/Framework/Cache/CacheWarmer/Product/ProductRouteWarmer.php

```diff
@@ -34,10 +35,20 @@ public function createMessage(SalesChannelDomainEntity $domain, ?array $offset):
         $iterator = $this->iteratorFactory->createIterator($this->definition, $offset);
         $query = $iterator->getQuery();
         $query
+            ->innerJoin(
```

Comment:
* Please provide tests
* Please prevent SQL short hands like `pv`, `p`, `pp`. Simply use the name of the table.